Documentation/doc/Documentation/License.txt

```diff
@@ -69,36 +67,16 @@ To enable checking, users have to define one of the following macros:
 The license checking is not a mean to control users as no information
 is collected or transmitted.
 
-\section seccgal_version Identifying the Version of CGAL
-
-Every release of \cgal defines the following preprocessor macros:
-
-<DL>
-<DT>`CGAL_VERSION_STR`</DT>
-<DD>a textual description of the current release (e.g., or 3.3 or 3.2.1 or 3.2.1-I-15) as a string literal</DD>
-<DT>`CGAL_VERSION_NR`</DT>
-<DD>a numerical description of the current release such that more recent
-releases have higher number.
-
-More precisely, it is defined as `1MMmmbiiii`, where `MM` is
-the major release number (e.g. 03), `mm` is the minor release
-number (e.g. 02), `b` is the bug-fix release number (e.g. 0),
-and `iiii` is the internal release number (e.g. 0001). For
-public releases, the latter is defined as 1000.  Examples: for the
-public release 3.2.4 this number is 1030241000; for internal release
-3.2-I-1, it is 1030200001.  Note that this scheme was modified around
-3.2-I-30.
-</DD>
-<DT>`CGAL_VERSION_NUMBER(M,m,b)`</DT>
-<DD>
-a function macro computing the version number macro from the
-M.m.b release version. Note that the internal release number is
-dropped here. Example: `CGAL_VERSION_NUMBER(3,2,4)` is equal to
-1030241000.
-</DD>
-</DL>
-
-The macro `CGAL_VERSION` is deprecated. It is the same as
-`CGAL_VERSION_STR`, but not as a string literal.
+\section licensesWhere Determining the license of a CGAL file
+
+It is specified in each file of the \cgal library which license applies to it.
+
+The \cgal documentation is distributed under the Creative Commons
+<a href="https://creativecommons.org/licenses/by-nc-nd/2.0/">BY-NC-ND</a> License.
+
+All files that do not have an explicit copyright or license notice (e.g., all
+examples, some demos, data files without a license, ...) are licensed under a very permissive license.
```

Comment:
Here should refer to the [CC0 ](https://creativecommons.org/publicdomain/zero/1.0/legalcode)and remove the `LICENSE.FREE_USE`

Reply:
@MaelRL this closes the discussion.